dts/bindings/stepper/ramp-parameters.yaml

```
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Jilay Sandeep Pandya
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Collection of ramp controller bindings for stepper motors.
```

```suggestion
Collection of ramp controllers for stepper motors.
```

It's in the `bindings` directory.

---

done :)